doc/migrating-configs.md

```diff
@@ -2,6 +2,41 @@
 
 Occasionally, there are changes made to Ignition's configuration that break backward compatibility. While this is not a concern for running machines (since Ignition only runs one time during first boot), it is a concern for those who maintain configuration files. This document serves to detail each of the breaking changes and tries to provide some reasoning for the change. This does not cover all of the changes to the spec - just those that need to be considered when migrating from one version to the next.
 
+## From Version 2.2.0 to 2.3.0
+
+There are not any breaking changes between versions 2.2.0 and versions 2.3.0 of the configuration specification. Any valid 2.2.0 configuration can be updated to a 2.3.0 configuration by simply changing the version string in the config.
+
+The 2.3.0 version of the configuration is greatly improved over version 2.2.0, with many new fields and behaviors added to the specification.
```

Comment:
This reads like an advertisement that we carry across bumps, but doesn't really add any information. Perhaps we can drop it?

Reply:
Fixed.